src/users/components/UserList.vue

```diff
@@ -1,10 +1,21 @@
 <template>
   <div class="list-wrapper">
     <QList>
+      <QItem
+        v-if="users.length > 1"
+        v-t="{ path: 'GROUP.MEMBERLIST', args: { total: users.length } }"
+        class="text-caption"
+      >
+        {{ activeUsersCount }}
```

Comment:
`activeUsersCount` no longer refers to anything, so can be removed! The `QItem` can become self-closing tag now, e.g. `<QItem ...props and stuff... />`

Reply:
Indeed. That was the most obvious of my hasty mistakes yesterday... :sweat_smile: